test-requirements.txt

```diff
@@ -1,4 +1,4 @@
-juju==3.2.0.0
+juju==3.2.0.1
```

Comment:
Do we need that specific version ? Why are we binding to that minor version ?

Reply:
Good question :) There was probably some bug in the lib, which made us pin the version. 
Not needed now.
Removed.